### ISSUE TITLE
Start task per URL not working in new browser session

### ISSUE THREAD

**Issue by Offerel** [roundcube/roundcubemail#9163]
Normally when a Roundcube session is started, the task can be selected by URL. For example to start the addressbook, i can call roundcube with ?_task=addressbook. This works only, within a already opened browser session. But if a new session is opened, for example with a fresh started browser window, the user lands always in the mail window. The url task parameter seems to be ignored.

At least for me, this looks like a bug or at least unexpected behavior. Can someone please fix this?

**Comment by alecpl** [comment]
Indeed. It used to work, so it must be some regression.

**Comment by Offerel** [comment]
Maybe it is a Cookie issue or some Info in the server session is not as expected or at least has changed. If you logg out from a Roundcube session and don't close the browser and reopen a URL with a given task, it is working as expected and opens the task.

As of yet, i have not looked at the login process that much, where it parses the url. But it should be not to difficult to find this minor issue. Many thx for all your work.

**Comment by alecpl** [comment]
I'm unable to reproduce the issue anymore. What Roundcube version and plugins?

**Comment by Offerel** [comment]
Indeed. I just tried this and can't reproduce the buggy behavior anymore either. Supposedly it was fixed with the update to 1.6.4. Besides my own plugins, only some standard plugins and "persisent_login" are active:

```
$config['plugins'] = [
	'caldavzap',
	'archive',
	'zipdownload',
	'persistent_login',
	'enigma',
	'hide_blockquote',
	'newmail_notifier',
	'subscriptions_option',
	'primitivenotes',
	'pictures',
	'storage',
	'contextmenu',
	'carddav',
	'elasticext',
	'identity_smtp'
];
```
In the meantime, very little else has changed on the server. I only changed to Debian 12 and with it nGinx. Additionally I have now activated the Brotli compression.

It works with all these plugins from the above list. I had first suspected a bug in my own plugins after I wanted to share a note via URL. On further analysis without plugins, I then noticed that this also affects contacts as a task. So from my point of view you can close the issue. Thanks for your help.
